platform/ios/MGLCategoryLoader.m

```diff
@@ -4,7 +4,16 @@
 #import "NSProcessInfo+MGLAdditions.h"
 #import "NSString+MGLAdditions.h"
 
+#import "MGLMapView.h"
+
 // https://github.com/mapbox/mapbox-gl-native/issues/2966
+// https://github.com/mapbox/mapbox-gl-native/issues/3113
+
+@interface MGLMapView (MGLCategoryLoader)
+
++ (void)mgl_linkMapView;
```

Comment:
It should be possible to just call an NSObject class method. For example, `[MGLMapView description];`.